pkgs/games/openra/builds.nix

```diff
@@ -0,0 +1,34 @@
+pkgs:
+
+let
+  f = import ./common.nix;
+  common = f (builtins.intersectAttrs (builtins.functionArgs f) pkgs // {
+    inherit (pkgs) stdenv;
```

Comment:
This line can be dropped?

Reply:
Yes it can. It was necessary at some point, but due to refactoring it no longer is, I will remove it.